devel/jsonnet-bundler/Portfile

```diff
@@ -3,84 +3,118 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/jsonnet-bundler/jsonnet-bundler 0.4.0 v
+go.setup            github.com/jsonnet-bundler/jsonnet-bundler 0.5.1 v
 categories          devel
-maintainers         {@danieltrautmann danieltrautmann.com:me} openmaintainer
+maintainers         {@0x1DA117 danieltrautmann.com:me} openmaintainer
 license             Apache-2
 
 description         A jsonnet package manager
 
 long_description    The jsonnet-bundler is a package manager for Jsonnet.
 
-build.pre_args      -o ${worksrcpath}/jb
-build.args          ./cmd/jb
+homepage            https://github.com/jsonnet-bundler/jsonnet-bundler
+
```

Comment:
The `github.setup` procedure from the github portgroup (which is invoked automatically by the `go.setup` procedure from the golang portgroup when invoked about a project hosted at GitHub) sets `homepage` for you automatically.
```suggestion
```

Reply:
@ryandesign Thanks for the information. I'll keep that in mind next time.
@reneeotten Thanks for taking care of it.